src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -1049,20 +1064,9 @@ public override int GetValues(object[] values)
         /// </remarks>
         /// <param name="ordinal">The zero-based column ordinal.</param>
         /// <returns>The value of the specified column.</returns>
+#pragma warning disable 618
```

Comment:
nit: can disable the warning for the entire block of obsoleted methods, rather than disabling/restoring multiple times